---
title: "Jacob Arie's Resume"
author: Jacob Arie van Diermen
date: "`r Sys.Date()`"
output:
  pagedown::html_resume:
    # set it to true for a self-contained HTML page but it'll take longer to render
    self_contained: true
# uncomment this line to produce HTML and PDF in RStudio:
knit: pagedown::chrome_print
---

# Aside

![Jacob Arie](assets/jacob_arie_cv.png){width="80%"}

## Contact Info {#contact}

- <i class="fa fa-envelope"></i> [jacob.arie.van.diermen\@ki.se](mailto:jacob.arie.van.diermen@ki.se){.email}
- <i class="fa fa-envelope"></i> [jarivandiermen\@hotmail.com](mailto:jarivandiermen@hotmail.com){.email}
- <i class="fa fa-globe"></i> [Jacob-Arie-van-Diermen-KI](https://ki.se/en/people/jacob-arie-van-diermen)
- <i class="fa fa-github"></i> [github.com/Suirotras](https://github.com/Suirotras)
- <i class="fa-brands fa-docker"></i> [hub.docker.com/u/jarivdiermen](https://hub.docker.com/u/jarivdiermen)
- <i class="fa-brands fa-linkedin"></i> [linkedin.com/in/jari-van-diermen](https://www.linkedin.com/in/jari-van-diermen-99a49712a/)
- <i class="fa fa-phone"></i> +31 653 270 827
- For more information, please contact me via email.

## Skills {#skills}

#### Languages

- <i class="fa-solid fa-circle"></i> Fluent Dutch (Native language)
- <i class="fa-solid fa-circle"></i> Fluent English (C2 level)
- <i class="fa-solid fa-circle"></i> Basic swedish reading level (A2 reading level)

#### Programming languages

- <i class="fa-solid fa-circle"></i> R
- <i class="fa-solid fa-circle"></i> Python
- <i class="fa-solid fa-circle"></i> Bash
- <i class="fa-solid fa-circle"></i> SLURM

# Main

## Jacob Arie van Diermen {#title}

### MSc, Research assistant

### [Karolinska Institutet](https://ki.se/en/startpage)


## Education {data-icon="graduation-cap" data-concise="true"}

### Utrecht university, Graduate School of Life Sciences

MSc Molecular and Cellular life sciences (MCLS), specialization Biophysics &
Molecular Imaging, Bioinformatics minor profile

Utrecht, The Netherlands

2023 - 2020

**Final grade: 8.02/10**

Bioinformatics minor profile

- *Thesis: Comparative genomics approach finds DNA-repair and immune-related genes
in the regenerative African spiny mouse*

Major research project

- *Thesis: Characterization of CSPP1 interaction with microtubules in cells*

### Utrecht university, Department of Biology

BSc in Biology

Utrecht, The Netherlands

2020 - 2017

**graduated with honors (cum laude)**

**Final grade: 8.14/10**

Bachelor thesis: Promising molecular targets for therapeutic intervention in
Alzheimer’s disease

## Research Experience {data-icon="laptop"}

### Research assistant

Karolinska Institutet, Clinical Neurosciences (Robert Harris Group)

Solna, Sweden

Current - 2023/07

<br>

- Continuation of the research project as a research assistant
- Preparing and writing a manuscript for publication
- Presenting data at lab meetings
- Creating Docker and Singularity containers for the positive selection
pipeline
- Working with positive selection tools like HyPhy MEME. HyPhy aBSREL and
PAML codeml
- Experience working with the Linux operating system (Ubuntu, Pop!_OS)
- Writing parallelized bash and R scripts for the positive selection pipeline
- Accessing and working with publicly available single-cell datasets form GTEX

### Teaching assistant

Utrecht university, Department of Biology

Utrecht, The Netherlands

2023/05 - 2023/02

<br>

Teaching assistant for the course `Kwantitatieve biologie (B-B1KWBI20)`,
a quantitative biology course for first year biology students. The course
consists of a series of lectures and practicals in which students learn
to use R for data analysis and statistics. As a teaching assistant, I supervised
and the students during the practicals sessions.

### Research intern

Karolinska Institutet, Clinical Neurosciences (Robert Harris Group)

Solna, Sweden

2023/03 - 2022/05

<br>

In my second year of my master ‘Molecular and Cellular life sciences’, I went to Stockholm for a 7 
month internship position at Karolinska institute. This internship was part of my bioinformatics 
minor profile. At the Center for molecular medicine (CMM), and under the supervision of
Sebastian Lewandowski, I performed a comparative genomics analysis which identified
genes under positive selection in the regenerative African spiny mice (Acomys cahirinus), and attempting
to link these genes to the regenerative phenotype. During this analysis, I build an independent analysis
pipeline from scratch, using available software and functions in R, python and bash. This analysis pipeline
was then executed on a large amount of genomics data, facilitated by a High performance computing (HPC) cluster.
I see this internship as a big part in my development as a bioinformatician.

Skills aqcuired:

- Bioinformatic analysis
- Comparative genomics
- Working with positive selection tools like HyPhy MEME and HyPhy aBSREL
- Working with genomics data
- Gene ontology analyses
- Cytoscape and STRINGdb network analysis
- Adept programming experience in R, Python and bash
- Experience with High performance computing in a SLURM environment
- Construction of a custom positive selection analysis pipeline for a large number of
multiple sequence alignments
- Experience with Git and github
- Presenting and writing skills
- Creation of a scientific poster for a conference
- Creation of data visualizations using R

### Research intern

Utrecht university, Cell biology, neurobiology and biophysics,
Anna Akhmanova Group (Cellular dynamics)

Utrecht, The Netherlands

2022/03 - 2021/01

<br>

In my first year of the master ‘Molecular and Cellular life sciences’, I had an 9-month internship in 
Anna Akhmanova’s Lab which specializes in microtubule dynamics, under the supervision of 
Cyntha van den Berg. During these 9 months, I investigated the characteristics of CSPP1, a ciliary and 
centrosomal protein, and its effect on microtubule dynamics. Using immunofluorescence 
experiments and fluorescence microscopy, I tried to understand the effect of different domains of 
this protein on microtubule dynamics. For example, one domain of CSPP1 is important for 
microtubule binding, while another domain is crucial for microtubule stabilization. Furthermore, 
using Expansion microscopy I tried to investigate whether this protein is a microtubule inner 
protein (MIP), meaning that it binds to the inside of the microtubule. Finally, using live cell 
imaging and photoactivatable GFP, I examined the live-recruitment of this protein to the 
microtubules.

Skills acquired:

- Extensive cell culture with mammalian adherent cell lines (COS7, RPE, HeLa)
- Molecular cloning with SNAPtag, LEXY, PA-GFP, EB3 constructs
- Transfections
- Transformation of *E.coli*
- Western blots (Semi-dry blotting)
- PCR
- Immunofluorescence staining
- Fluorescence microscopy
- Expansion microscopy (STED)
- Live-cell imaging (spinning disk, TIRF)
- Experiment planning
- Presenting data
- General lab skills

## <i class="fa-solid fa-box-archive"></i> Additional skills {data-icon=NULL}

### Programming languages

N/A

N/A

<br>

- R (Experience in base R and a large number of R packages)
- Python (Experience in base python and popular modules like numpy, pandas,
matplotlib and machine learning modules like scikit-learn). Also experience with
python single cell libraries like scanpy and anndata.
- Bash (Experience in writing bash scripts for bioinformatics pipelines)
- SLURM (Performing a large number of analyses using SLURM on HPC environments)

### Software skills

N/A

N/A

- STRING network analysis
- gene ontology analysis
- Cytoscape
- Inkscape
- Docker
- Singularity
- Git
- Github
- Rsudio
- Anaconda
- Visual studio code
- Linux (Ubuntu, Pop!_OS)
- HyPhy (aBSREL, MEME)
- PAML codeml
- Polyphen-2
- MS word, Excel, PowerPoint
- GraphPad Prism 8
- Fiji/ ImageJ
- Biorender
- Reference managers (E.g. Mendeley, Zotero)

## Talks and conferences {data-icon="person-chalkboard"}

### Genetic deconvolution of regenerative phenotypes in Acomys cahirinus

Tissue and Motion Conference 2022 (Poster)

Djurhamn, Sweden

2022

**Aditya Singh**, Sebastian A. Lewandowski, Katarina Tengvall, Michael Dong,
Kerstin Lindblad-Toh

## Recognition {data-icon="award"}

### Theme winner (Energy) Sustainable campus challenge 2020

Utrecht Challenge Alliance

Utrecht, The Netherlands

2020

<br>

I participated in the sustainable campus challenge 2020 organized by the Utrecht challenge alliance. This is a partnership for open innovation to find regional solutions to social issues in cooperation between companies, public organisations, governments and educational institutions. We were tasked to come up with a concept to reduce the energy usage of student housing and came up with a concept that rewards students for their sustainable behaviour. Our concept was chosen as the best concept to reduce student energy usage.

### Latin honors (Cum laude) for BSc in Biology

Utrecht university, Department of Biology

Utrecht, The Netherlands

2020
